lib/changelog_web/views/time_view.ex

```diff
@@ -62,10 +64,13 @@ defmodule ChangelogWeb.TimeView do
   end
 
   def rss(ts) when is_nil(ts), do: ""
-  def rss(ts) do
-    {:ok, result} =
-      ts
-      |> Timex.format("{RFC1123}")
+  def rss(ts), do: ts |> format_to("{RFC1123}")
+
+  def to_rfc3339(ts) when is_nil(ts), do: ""
```

Comment:
A minor nitpick, but I'd prefer simplifying the name to just `rfc3339` instead of `to_rfc3339` variant. The reason being I think of these functions as providing a representation of time in a specified format or purpose. When you prefix the `to_` it looks more like a type conversion.